mxcubeweb/core/adapter/beamline_action_adapter.py

```diff
@@ -1,21 +1,39 @@
+from typing import ClassVar
+
 import logging
 from enum import Enum
 
-from mxcubeweb.core.adapter.adapter_base import ActuatorAdapterBase
+from mxcubecore.HardwareObjects.BeamlineActions import BeamlineActions
+from mxcubecore.BaseHardwareObjects import HardwareObject
+
+from mxcubeweb.core.models.configmodels import AdapterResourceHandlerConfigModel
+
+from mxcubeweb.core.adapter.adapter_base import AdapterBase
 from mxcubeweb.core.models.adaptermodels import (
     HOActuatorValueChangeModel,
     NStateModel,
-    StrValueModel,
 )
 
+resource_handler_config = AdapterResourceHandlerConfigModel(
+    commands=["stop"], attributes=["data"]
+)
+
+
+class BeamlineActionAdapter(AdapterBase):
+    SUPPORTED_TYPES: ClassVar[list[object]] = [BeamlineActions]
 
-class BeamlineActionAdapter(ActuatorAdapterBase):
-    def __init__(self, ho, *args, **kwargs):
+    def __init__(
+        self,
+        ho: HardwareObject,
+        role: str,
+        app,
+        default_resource_handler_config: AdapterResourceHandlerConfigModel = resource_handler_config,  # noqa: E501
```

Comment:
So I'm not a fan of this, but I did not find any way of splitting this line

Reply:
I guess it's fine to put an exception.

Otherwise I guess the most common solution is like this:

```suggestion
        default_resource_handler_config: AdapterResourceHandlerConfigModel = (
            resource_handler_config
        ),
```

Reply:
I also believe the exception here is fine, but I'm also fine with @fabcor-maxiv suggestion.

I just saw that the nomenclature for `default_resource_handler_config` and `resource_handler_config` is the other way around in the base class (`adapterbase`). It's not a big issue, but it might cause confusion in the future.

Reply:
Thanks